src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalVariableNameCheck.java

```diff
@@ -96,10 +96,11 @@
  * <p>
  * An example of how to configure the check to allow one character variable name in
  * <a href="https://docs.oracle.com/javase/tutorial/java/nutsandbolts/for.html">
- * initialization expressions</a> in FOR loop:
+ * initialization expressions</a> in FOR loop, where regexp allows 2 or more chars:
  * </p>
  * <pre>
  * &lt;module name="LocalVariableName"&gt;
+ *   &lt;property name="format" value="^[a-z][_a-zA-Z0-9]+$"/&gt;
```

Comment:
please do us a favor to update documentation a bit more.
Please make it like:
`An example of how to configure the check ...`
following with
```
Code Example:
class MyClass {
  void MyMethod() {
    int goodName = 0;
    int i = 1; // violation
    for (int var = 1; var < 10; var++) { //OK
      int j = 1; // violation
    }
  }
}
```

xdoc should also be update, it should match javadoc.

Reply:
Current example shows case when regex allows 2+ chars, and property is used to make an exception.

Did you mean to add one more example with allowOneCharVarInForLoop = false and regex for 3+ chars or adjust existing one?

Reply:
I added one more example. Please take a look.